src/main/java/com/github/princesslana/eriscasper/ErisCasper.java

```diff
@@ -38,21 +40,22 @@
   private final Payloads payloads = new Payloads(jackson);
 
   private final Routes routes;
+  private final Optional<Shard> shard;
 
-  private ErisCasper(BotToken token) {
+  private ErisCasper(BotToken token, Optional<Shard> shard) {
```

Comment:
Should we avoid this optional by having using a Shard of [0,1] as a default?

Reply:
I think having no sharding is a bit safer, I dont think telling discord we are using shard [0,1] is useful and Optional makes sense and looks like what it does.